python/python_api/cloe/_runner.py

```diff
@@ -177,6 +178,7 @@ def log_level(self, value):
 
     def bind_plugin_types(self, lib: Path):
         import importlib
+        import importlib.util
```

Comment:
I only see `imnportlib.util` being used. In that case, you can just remove the line above and have only one import.

I would also while we're making changes, move the `sys` import to the top of the file.